Sources/StreamChatUI/CommonViews/Attachments/AttachmentViews/VideoAttachmentComposerView.swift

```diff
@@ -0,0 +1,113 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import AVKit
+import StreamChat
+import UIKit
+
+/// A view that displays the video attachment preview in composer.
+public typealias VideoAttachmentComposerView = _VideoAttachmentComposerView<NoExtraData>
+
+/// A view that displays the video attachment preview in composer.
+open class _VideoAttachmentComposerView<ExtraData: ExtraDataTypes>: _View, ThemeProvider {
```

Comment:
The name feels confusing, what's the Image counterpart's name? I think of something like `VideoAttachmentComposerThumbnailView`

Reply:
> what's the Image counterpart's name

It's `ImageAttachmentView` but the component name is `messageComposerImageAttachmentView` 🤔 Would be nice to agree upon video/image views naming (both for 3 places: **composer/message cell/gallery**). 

The `*ThumbnailView` works for me 👍 The shorter `*Preview` also does (e.g. `VideoAttachmentComposerPreview`). 

So it can be:
`*AttachmentComposerThumbnailView/*AttachmentComposerPreview`
`*AttachmentCellThumbnailView/*AttachmentCellPreview`
`*AttachmentCell`

Reply:
All the Component's config names of the composer are quite confusing right now. The name should not be `messageComposerImageAttachmentView` but only `imageAttachmentView`. The thing is that we might use these views in places other than the composer. We probably should just define something like `previewImaageAttachmentView` and `thumbnailImageAttachmentView`... I don't know, we def need to re-think these components views names

Reply:
maybe `_VideoAttachmentComposerPreviewView` ?

Reply:
> All the Component's config names of the composer are quite confusing right now. The name should not be `messageComposerImageAttachmentView` but only `imageAttachmentView`. The thing is that we might use these views in places other than the composer. We probably should just define something like `previewImaageAttachmentView` and `thumbnailImageAttachmentView`... I don't know, we def need to re-think these components views names

the chances the `*AttachmentComposerPreviewView/Preview/ThumbnailView` will be used somewhere else are low while having a generic name makes it tricky to understand where a component is used 🤔

Components like `loadingIndicator` are general purpose and should no doubt have a generic naming but with attachment previews we'd better be concrete IMHO. Otherwise, it's gonna be super confusing for customers to distinguish between attachment components by name (for image/video attachment there're 3 different ones) 😄

Reply:
✅ Discussed internally and renamed